src/web/layouts/InteractiveImmersiveLayout.tsx

```diff
@@ -196,7 +196,9 @@ export const InteractiveImmersiveLayout = ({
 
 	return (
 		<>
-			<Global styles={interactiveGlobalStyles} />
+			{CAPI.isLegacyInteractive && (
+				<Global styles={interactiveGlobalStyles} />
```

Comment:
I wonder if renaming this to `interactiveLegacyStyles` would be helpful, too.

Reply:
Good shout

Reply:
Done :D